<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APMShortcutModesDialog</class>
 <widget class="QDialog" name="APMShortcutModesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Shortcut Mode Actions</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>44</y>
     <width>341</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="opt1ModeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="opt2ModeComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>8</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="opt3ModeComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>8</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="opt4ModeComboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>8</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>APMShortcutModesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>APMShortcutModesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
